main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -40,9 +38,8 @@
 public class Main {
 
     public static void main(String[] argv) {
-        Logger logger = LogManager.getLogger();
         Arguments args = new Arguments();
-        CliParametersAnalyzer cliParametersAnalyzer = new CliParametersAnalyzer(logger);
+        CliParametersAnalyzer cliParametersAnalyzer = new CliParametersAnalyzer();
```

Comment:
Just reading Flogger's [docs best practices](https://google.github.io/flogger/best_practice) 👀📖

> Don’t pass loggers between classes
> Only log via a single static logger instance that’s created inside the same source file as the log statement. There is generally no reason to pass a logger from one class to another.
> 
> Why do people think that loggers should be passed between classes?
> I think the primary reason is that people misunderstand where the class name in log output comes from. It is not based on the name of the logger; it’s the location of the log statement, regardless of the name of the logger used.
> 
> The logger instance does affect the configuration of the logger, but if all loggers use the same configuration (which is very common) then different logger instances will behave identically and there no reason to pass them around.

Thanks Alexej, it's good to know!